packages/blocks/src/api/raw-handling/phrasing-content-reducer.js

```diff
@@ -4,7 +4,9 @@
 import { wrap, replaceTag } from '@wordpress/dom';
 
 export default function( node, doc ) {
-	if ( node.nodeName === 'SPAN' ) {
+	// In jsdom-jscore, 'node.style' can be null.
```

Comment:
Kinda a nit but, perhaps it's not too clear what `jsdom-jscore` means in this context. Of course, jsdom-jscore it's the DOM parser we're using in the native mobile (React Native) version of Gutenberg, but I think I'd suggest add that bit to the comment to, to help out web-side friends read the code more easily.

So, my suggestion is to refer with `In native mobile (jsdom-jscore), ...` instead of the current. What do you think? Same for a couple other similar references in this same PR.

Reply:
On a less of a nit thing, I wonder about the (emphasis mine on the "can"):

```
'node.style' *can* be null.
```

Does that mean that sometimes node.style is undefined and some other it's not?

In the cases where it's undefined, is it OK to just ignore it like we're doing currently in the PR? I wonder if being undefined denotes an unrecoverable error and the we should let the user know about it, in some way.

Should we at least log an error/warning?

Reply:
Yes, good suggestion! I'll edit those, and keep that in mind for future comments.

Reply:
> Does that mean that sometimes node.style is undefined and some other it's not?

With the change of wording in the comment to: `In native mobile (jsdom-jscore), ...`, it's probably more accurate to say that `node.style` *is* undefined.

> Should we at least log an error/warning?

In the codepath where this is used, `node` is actually an `HTMLElement`, as the condition is satisfied that the element's `nodeName` is `SPAN`. HTMLElement nodes are supposed to have a `style` property which returns a read-only [`CSSStyleDeclaration`](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration) object, which can be used to retrieve the inline styles applied to the element via the style attribute. This is then used to wrap the node in appropriate tag(s) (`strong`, `em`, etc.). In the case where the DOM implementation does not support the `style` property on a node, I believe it is safe to ignore, but that we may lose style information that was associated with the content via inline styles on a span's attribute.

The property seems to have had wide support for a long time, but I wonder if there are any browsers that don't support this property, or that leave it undefined or null when no style attribute was present on the element. My reason for wording the comment somewhat "vaguely" is that I tend to assume that there could be such a case in the wild, and thus the safety check could be useful even in the browser implementation. I checked the [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style#Browser_compatibility) for it on MDN, and it lists a few question marks, but that just means unconfirmed AFAIK.

**TL;DR:**

The check will not lead to an unrecoverable error, but will gracefully degrade the feature that normalizes some inline styles on span tags. I don’t have a strong opinion on whether that warrants logging a warning; I suppose it depends on the verbosity desired (e.g. web: “please use a better browser :)”, mobile: "mobile currently lacks support for this feature").

Reply:
> mobile: "mobile currently lacks support for this feature"

I think that's a good case to actually log since, these things are useful while offering support to users.

While at it, since this is a known limitation, let's add a "Known Limitations" section to https://github.com/wordpress-mobile/gutenberg-mobile/pull/553 and mention this there. WDYT?